### PR TITLE
SIP-371: Add sUSDe LP collateral support to Synthetix V3 on Arbitrum

### PR DESCRIPTION
**Pull request by noisekit**
- revert of #274
- un-revert of #241
- minor adjustments of naming/casing/style to match current convention

### CODE REVIEW

File: tomls/omnibus-arbitrum-mainnet/rewards/seth_perps_liquidations_rewards.toml [hunk 1]
```diff
@@ -21,6 +21,7 @@ args = [
         "<%= settings.arb_address %>",
         "<%= settings.usdc_address %>",
         "<%= settings.usde_address %>",
+        "<%= settings.susde_address %>",
```
Comment (noisekit) [inline tomls/omnibus-arbitrum-mainnet/rewards/seth_perps_liquidations_rewards.toml:24]: NOTE I am intentionally not registering rewards distributor to catch this with #426 test

Comment (noisekit) [reply tomls/omnibus-arbitrum-mainnet/rewards/seth_perps_liquidations_rewards.toml:24]: Good. Tests are failing as expected. Will add missing registrations shortly.


image

File: tomls/omnibus-arbitrum-sepolia/collaterals/susde.toml [hunk 2]
```diff
@@ -1,17 +1,9 @@
...
-[setting.sUSDe_liquidation_ratio]
-defaultValue = "<%= parseEther('1.05') %>"
-
-[setting.sUSDe_liquidation_reward]
-defaultValue = "<%= parseEther('1') %>"
-
-[setting.sUSDe_min_delegation]
-defaultValue = "<%= parseEther('100') %>"
+[var.susde_collateral_config]
+susde_max_collateral_limit = "<%= parseEther(String(3_000_000)) %>"
```
Comment (noisekit) [inline tomls/omnibus-arbitrum-sepolia/collaterals/susde.toml:2]: as per sccp 356 we expect this value to be 10 mil.

File: tomls/omnibus-arbitrum-mainnet/collaterals/susde.toml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+[var.susde_collateral_config]
+susde_max_collateral_limit = "<%= parseEther(String(3_000_000)) %>"
```
Comment (kaleb-keny) [inline tomls/omnibus-arbitrum-mainnet/collaterals/susde.toml:2]: can you please update to 10m https://sips.synthetix.io/sccp/sccp-354/ I understand not spelled out in the sccp, but the max amount on susde should be 10, on synth or the underlying

File: tomls/omnibus-arbitrum-mainnet/collaterals/susde.toml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+susde_issuance_ratio = "<%= parseEther('2.00') %>"
```
Comment (kaleb-keny) [inline tomls/omnibus-arbitrum-mainnet/collaterals/susde.toml:3]: can you please increase this to 1000% https://sips.synthetix.io/sccp/sccp-352/

File: tomls/omnibus-arbitrum-sepolia/collaterals/susde.toml (hunk 2 above)
Comment (kaleb-keny) [reply tomls/omnibus-arbitrum-sepolia/collaterals/susde.toml:2]: bump on this 😅  not updated

File: tomls/omnibus-arbitrum-mainnet/collaterals/susde.toml (hunk 3 above)
Comment (noisekit) [reply tomls/omnibus-arbitrum-mainnet/collaterals/susde.toml:2]: done

File: tomls/omnibus-arbitrum-mainnet/collaterals/susde.toml (hunk 4 above)
Comment (noisekit) [reply tomls/omnibus-arbitrum-mainnet/collaterals/susde.toml:3]: done

File: tomls/omnibus-arbitrum-sepolia/collaterals/susde.toml [hunk 5]
```diff
@@ -1,17 +1,9 @@
...
+susde_issuance_ratio = "<%= parseEther('2.00') %>"
```
[the remark this answers is not in the document]
Comment (noisekit) [reply tomls/omnibus-arbitrum-sepolia/collaterals/susde.toml]: done

File: tomls/omnibus-arbitrum-sepolia/collaterals/susde.toml (hunk 2 above)
Comment (noisekit) [reply tomls/omnibus-arbitrum-sepolia/collaterals/susde.toml]: done

**Review by kaleb-keny — APPROVED** [review]

**Pull request merged**